.circleci/config.yml

```diff
@@ -288,7 +288,7 @@ workflows:
   build:
     when: << pipeline.parameters.run_default_workflow >>
     jobs:
-      - unit-php74-magento24
+      - integration-php74-magento24
       - unit-php72-magento23
       - unit-php71-magento22
       - unit-php70-magento22
```

Comment:
we should also remove PHP7.0 tests, as it is no longer supported by Magento
PHP7.1 and PHP7.2 are no longer recommended as well but may keep them for now

Reply:
@danac-gs yes, this PR is still WIP, I will let you know once it is ready. Thanks

Reply:
@dumega for WIP you can create a draft PR, or convert it to draft as in the picture
![image](https://user-images.githubusercontent.com/37150717/107369064-967ea480-6ae1-11eb-8ae3-4f808e789a72.png)